### DISCUSSION TITLE
Guess films

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#506] [Galaxy Trivia] [answered]
🎄☹️🟢
🐝🔍👨‍👩‍👧‍👦
🐁🧑‍🍳

**Comment by peterbaccaro** [comment]
1. THE GRINCH
2. BEE MOVIE
3. RATATOUILLE

**Reply by ParanoidUser** [reply]
So close! The movie in the second emoji is in the same genre as Bee Movie, but premiered 18 years earlier. Its success led to two sequels. Can you guess it? 🥤🍿

**Reply by peterbaccaro** [reply, marked as answer]
2. HONEY I SHRUNK THE KIDS.

**Reply by ParanoidUser** [reply]
Fantastic!

**Comment by thuongtruong1009** [comment]
🎄☹️🟢 - This icon combination represents the movie "The Grinch" which is a Christmas comedy film about a green grumpy creature who hates Christmas.

🐝🔍👨‍👩‍👧‍👦 - This icon combination represents the movie "Bee Movie" which is an animated comedy film about a bee who sues humans for stealing honey.

🐁🧑‍🍳 - This icon combination represents the movie "Ratatouille" which is an animated film about a rat named Remy who dreams of becoming a chef in a famous Parisian restaurant.

🥤🍿 is a commonly used icon combination to represent movies, but one film that immediately comes to mind is "The Bucket List" starring Jack Nicholson and Morgan Freeman. The film tells the story of two terminally ill men who escape from a cancer ward and embark on a road trip with a wish list of things to do before they die, including visiting the Taj Mahal and skydiving. The characters are shown enjoying 🥤🍿 while watching a movie together, which has become a memorable scene from the film.

**Reply by quicksen1** [reply]
Die Hard
